src/main/java/gov/loc/repository/bagit/hash/Hasher.java

```diff
@@ -3,38 +3,80 @@
 import java.io.BufferedInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
 import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.Formatter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gov.loc.repository.bagit.domain.Manifest;
 
 /**
  * Convenience class for generating a HEX formatted string of the checksum hash. 
  */
 public final class Hasher {
+  private static final Logger logger = LoggerFactory.getLogger(Hasher.class);
+  private static final int _64_KB = 1024 * 64;
```

Comment:
Not sure how much I care about this but I generally prefer to use symbolic names — e.g. `HASHER_BLOCK_SIZE` or `CHUNK_SIZE`? — so it's a little easier to understand at first glance and to minimize diff noise if this were to change in the future. That said, in this case it's probably unnecessary pedantry.